Source/player.cpp

```diff
@@ -1016,7 +996,7 @@ bool PlrHitMonst(int pnum, int m, bool adjacentDamage = false)
 	return true;
 }
 
-bool PlrHitPlr(int pnum, int8_t p)
+bool PlrHitPlr(Player &attacker, int8_t p)

```

Comment:
Unlike missile attacks this method allows for lifesteal effects and modifies the attacker, worth having a comment explaining why a non-const reference is used here.